modules/apps/remote-web-app/remote-web-app-web/src/main/resources/META-INF/resources/remote_web_app_admin/view.jsp

```diff
@@ -0,0 +1,130 @@
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/remote_web_app_admin/init.jsp" %>
+
+<clay:management-toolbar
+	actionDropdownItems="<%= remoteWebAppAdminViewManagementToolbarDisplayContext.getActionDropdownItems() %>"
+	clearResultsURL="<%= remoteWebAppAdminViewManagementToolbarDisplayContext.getClearResultsURL() %>"
+	componentId="remoteWebAppAdminViewManagementToolbar"
+	creationMenu="<%= remoteWebAppAdminViewManagementToolbarDisplayContext.getCreationMenu() %>"
+	disabled="<%= remoteWebAppAdminViewManagementToolbarDisplayContext.isDisabled() %>"
+	itemsTotal="<%= remoteWebAppAdminViewManagementToolbarDisplayContext.getTotal() %>"
+	searchContainerId="remoteWebAppEntries"
+	selectable="<%= true %>"
+	showSearch="<%= true %>"
+/>
+
+<clay:container-fluid>
+	<aui:form action="<%= currentURL %>" method="get" name="fm">
+		<aui:input name="<%= Constants.CMD %>" type="hidden" />
+		<aui:input name="redirect" type="hidden" value="<%= currentURL %>" />
+
+		<liferay-ui:search-container
+			id="remoteWebAppEntries"
+			searchContainer="<%= remoteWebAppAdminViewManagementToolbarDisplayContext.getSearchContainer() %>"
+			total="<%= remoteWebAppAdminViewManagementToolbarDisplayContext.getSearchContainer().getTotal() %>"
+		>
+			<liferay-ui:search-container-results
+				results="<%= searchContainer.getResults() %>"
+			/>
+
+			<liferay-ui:search-container-row
+				className="com.liferay.remote.web.app.model.RemoteWebAppEntry "
+				keyProperty="entryId"
+				modelVar="entry"
+			>
+
+				<%
+				Map<String, Object> rowData = HashMapBuilder.<String, Object>put(
+					"actions", StringUtil.merge(remoteWebAppAdminViewManagementToolbarDisplayContext.getAvailableActions(entry))
+				).build();
+
+				row.setData(rowData);
+
+				PortletURL rowURL = renderResponse.createRenderURL();
+
+				rowURL.setParameter("mvcRenderCommandName", "/announcements/view_entry");
+				rowURL.setParameter("redirect", currentURL);
+				rowURL.setParameter("entryURL", String.valueOf(entry.getURL()));
+				%>
+
+				<liferay-ui:search-container-column-text
+					cssClass="table-cell-content"
+					href="<%= rowURL %>"
+					name="url"
+					value="<%= HtmlUtil.escape(entry.getURL()) %>"
+				/>
+
+				<liferay-ui:search-container-column-jsp
+					path="/remote_web_app_admin/entry_action.jsp"
+				/>
+			</liferay-ui:search-container-row>
+
+			<liferay-ui:search-iterator
+				markupView="lexicon"
+				searchContainer="<%= searchContainer %>"
+			/>
+		</liferay-ui:search-container>
+	</aui:form>
+</clay:container-fluid>
+
+<%--
```

Comment:
This may need to be recovered, once we implement the delete action. It was copied from `Announcements` portlet.